build/index.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <title>Главная страница</title><!--[if IE]>
+    <meta http-equiv="X-UA-Compatible" content="IE = edge"><![endif]-->
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="keywords" content="">
+    <link rel="icon" type="image/png" href="static/img/general/favicon.png">
+    <link rel="stylesheet" type="text/css" href="static/css/main.css"><!--[if lt IE 9]>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js"></script><![endif]-->
+  </head>
+  <body class="index-page">
+    <div class="wrapper">
+      <header>
+        <div class="header">
+          <div class="sandwich">
+            <input type="checkbox" id="menu__toggle">
+            <label class="menu-btn" for="menu__toggle"><span></span></label>
+            <nav class="header-nav"><a class="header-logo" href="#" data-to-elem="#product-info">back to the office</a>
+              <ul class="header-nav__list">
+                <li class="header-nav__item"><a class="header-nav__link" href="#" data-to-elem="#about">About us</a></li>
+                <li class="header-nav__item"><a class="header-nav__link" href="#" data-to-elem="#functions">Functions</a></li>
+                <li class="header-nav__item"><a class="header-nav__link" href="#" data-to-elem="#contacts">Contact Us</a></li>
+              </ul>
+            </nav>
+          </div>
+          <button class="g-btn g-btn--green open-popup">apply for</button>
+        </div>
+      </header>
+      <div class="content">
+        <section class="product-info" id="product-info">
+          <div class="container">
+            <div class="product-info__content">
+              <div class="product-info__left fadeInLeft wow" wow-delay="2s">
+                <div class="product-info__header">
+                  <div class="product-condition">soon</div>
+                  <div class="product-status">Under development</div>
+                </div>
+                <h1 class="main-title">Back to the Office<span class="product-status">Under development</span></h1>
+                <p class="description-text">The application for remote work.</p>
+                <p class="description-text">Back To The Office will help remote employees to become a part of the team.</p>
```

Comment:
please change  "Back To **T**he Office" to "Back To **t**he Office"

Reply:
fixed